### PR TITLE
Updated topics.json, exercises.json, contents.json, etc -- plus refactoring of contentload and friends to work with new DB-backed assessment items and changes made by KA

### PR DESCRIPTION
**Pull request by jamalex**
- Updated JSON data from KA to get latest content.
- init_assessment_items: bulk creation, and DB and JSON location options
- Rename DB backend, move AI DB to /content/khan/, and re-org some settings.
- Fixes to generate_assessment_zips (multi-folder, DB-backed, KA changes).
    - Put image files into multiple subfolders in /content/khan/ (by first 3
      characters of name)
    - Instead of assessmentitems.json and assessmentitems.json.version, now
      build and pack in assessmentitems.sqlite and assessmentitems.version
    - Fixes to properly replace KA image URLs, in response to their changes.
- Unpack the new assessments.zip format (with DB inside).
- Fixes to contentload to deal with the flat topic tree now expected.
- Fixes to contentload tests to work with the latest management commands.
- Gracefully return None if assessment item DB is not initialized.

### CODE REVIEW

**Comment by benjaoming** [comment]
:+1: 

Can we merge?

**Comment by jamalex** [comment]
Latest assessments.zip, for testing (with this branch only):
http://eslgenie.com/media/assessments.zip

File: kalite/contentload/management/commands/contentload.py [hunk 1]
```diff
@@ -162,12 +133,10 @@ def handle(self, *args, **options):
 
         sys.stdout.write(
```
Comment (aronasorman) [inline kalite/contentload/management/commands/contentload.py]: Can you use `logging` here?

File: kalite/contentload/management/commands/generate_assessment_zips.py [hunk 2]
```diff
@@ -17,18 +18,28 @@
...
+IMAGE_URL_REGEX = re.compile('https?://[\w\.\-\/]+\/(?P<filename>[\w\.\-\%]+\.(png|gif|jpg|jpeg|svg))', flags=re.IGNORECASE)
 
 WEB_GRAPHIE_URL_REGEX = re.compile('web\+graphie://ka\-perseus\-graphie\.s3\.amazonaws\.com\/(?P<filename>\w+)', flags=re.IGNORECASE)
 
+IMAGE_URLS_NOT_TO_REPLACE = set([
+    "http://www.dogs.com/photo.jpg",
+    "https://www.kasandbox.org/programming-images/creatures/OhNoes.png",
+])
+
+MANUAL_IMAGE_URL_TO_FILENAME_MAPPING = {
```
Comment (aronasorman) [inline kalite/contentload/management/commands/generate_assessment_zips.py]: Can you add a comment explaining what this variable is for?

File: kalite/topic_tools/__init__.py [hunk 3]
```diff
@@ -452,6 +453,8 @@ def get_assessment_item_data(request, assessment_item_id=None):
         assessment_item = main_models.AssessmentItem.objects.get(id=assessment_item_id)
     except main_models.AssessmentItem.DoesNotExist:
         return None
+    except DatabaseError:
+        return None
```
Comment (MCGallaspy) [inline kalite/topic_tools/__init__.py]: I know I'm late to the party here, but the intent is to handle the case when the db doesn't exist, yes? I wonder if `DatabaseError`s are raised in any other circumstances that we might not wish to catch here?

**Pull request merged**